src/blueprints/notifications.py

```diff
@@ -18,38 +21,41 @@
 Good luck!
 """
 
-@notifications.route('/check_for_predictions', methods=['POST'])
-def check_upcoming_games()
+@notifications.route('/check_for_predictions', methods=['GET'])
+def check_upcoming_games():
 
     timezone = pytz.timezone('Europe/London')
     today = datetime.today()
     today = datetime(today.year, today.month, today.day)
-    matches = session.query(Match).filter(Match.match_date = today).all()
+
+    matches = session.query(Match).filter(Match.match_date == today).all()
+    
     for match in matches:
         kick_off = getattr(match, 'kick_off_time')
         match_date = getattr(match, 'match_date')
         combined = datetime.combine(match_date, kick_off)
         combined = timezone.localize(combined)
-        
+
         current_time = datetime.now(timezone)
 
-        if combined < (current_time + datetime.timedelta(hours = 1)) and not check_kicked_off(match.matchid):
+        if combined < (current_time + timedelta(hours = 1, minutes=59)) and not check_kicked_off(match.matchid):
             check_user_prediction(match.matchid)
+    return jsonify({"message": "no games in the next hour"})
```

Comment:
This will always be returned as you're not returning the result from the 'check_user_prediction' call I think